etl/steps/data/garden/gapminder/2023-03-31/population.meta.yml

```diff
@@ -0,0 +1,35 @@
+all_sources:
+  - un_wpp: &gapminder_source
+      name: Gapminder (2022)
+      published_by: Gapminder, Population (v7) 2022
+      url: https://gapm.io/dl_popv7
+      date_accessed: "2023-03-31"
+      publication_date: "2022-10-19"
+      publication_year: 2022
+
+dataset:
+  title: Population (Gapminder, 2022)
+  description: |-
```

Comment:
Isn't this description identical to the one in snapshot? If so, then I suppose you don't need to write it again here.

Reply:
Very good point